Helper/Data.php

```diff
@@ -788,6 +788,33 @@ public function getAPIKey($storeId = null)
         return $apiKey;
     }
 
+    /**
+     * Retrieve the Client key
+     *
+     * @param null|int|string $storeId
+     * @return string
+     */
+    public function getClientKey($storeId = null)
+    {
```

Comment:
Not a big deal but I thing this function can be simplified.

```suggestion
    return trim(
          $this->getAdyenAbstractConfigData(
              $this->isDemoMode($storeId) ? 'client_key_test' : 'client_key_live',
              $storeId
          )
      );
```